### PR TITLE
Experiment: Github CI to build hirsute packages

### PR DESCRIPTION
**Pull request by oskirby**
This is a test to try and get automated Debian packages built for 21.04 hirsute by using `pbuilder` to create clean build environments for distributions which may differ from the base image provided by Github.

### CODE REVIEW

**Comment by oskirby** [comment]
The test seems to work! Although, only for distributions which can be built without relying on a PPA for the Qt libraries, so for the meantime this means just Ubuntu 21.04/hirsute.

File: .gitignore [hunk 1]
```diff
@@ -24,6 +24,7 @@ mozillavpn-daemon
 translations/*.qm
 translations/*.pri
 linux/netfilter/netfilter.[ah]
+linux/netfilter/vendor/
```
Comment (bakulf) [inline .gitignore:27]: There are issues with the content of this PR. This is unrelated, right?

**Review by bakulf — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (oskirby) [reply .gitignore:27]: This line in the `.gitignore` file is intended to be there. When we download the Go module's dependencies for offline building using the `go mod vendor`, it will download them all into this directory. We can treat it as generated content, so it shouldn't be committed to the repository.

**Review by bakulf — APPROVED** [review]
